index.html

```diff
@@ -2039,7 +2037,7 @@ <h2>Updated</h2>
       </p>
 
       <dl>
-          <dt><dfn>updated</dfn></dt>
+          <dt>updated</dt>
```

Comment:
I'd like to understand the thought process behind removing the dfns, here. One good argument for removing them is that we don't cite them or link to them from elsewhere in the specification? We may want to discuss why not before removing them. It would be nice to link to these items if we mention them elsewhere in the specification... that was the intent long ago, and then I never got around to writing the ReSpec extension to do that like we do for the `<a>reference</a>` stuff.

Reply:
Right, they're not cited anywhere

Reply:
I also find it a bit odd to have definitions around property names. In my head I differentiate between definitions for terms/concepts and the specific property names, and I think it makes more sense to link to the section in which they are defined than to put them in as term definitions.